### PR TITLE
Set the default device to Vega for HIP_backend

### PR DESCRIPTION
**Pull request by zjing14**

### CODE REVIEW

File: miopengemm/src/oclutil.cpp [hunk 1]
```diff
@@ -1040,10 +1041,13 @@ DevInfo::DevInfo(const cl_device_id& device_)
   initialise();
 }
 
-DevInfo get_fiji_devinfo() { return DevInfo("gfx803", 64); }
+DevInfo get_fiji_devinfo() { return DevInfo("gfx803", "gfx803", 64); }
+
+DevInfo get_vega_devinfo() { return DevInfo("gfx900", "gfx900", 32); }
```
Comment (dagamayank) [inline miopengemm/src/oclutil.cpp]: What does `32` do here at the end?

Comment (zjing14) [reply miopengemm/src/oclutil.cpp]: Based on the following code.

cl_set_device_info(device,
                     CL_DRIVER_VERSION,
                     info_st.size(),
                     &info_st[0],
                     &info_size,
                     "obtaining CL_DRIVER_VERSION",
                     strict);
  driver_version = info_st.substr(0, info_size - 1);

  if (platinfo.vendor.find("vidia") != std::string::npos ||
      platinfo.vendor.find("NVIDIA") != std::string::npos)
  {
    wg_atom_size = 32;
  }

  else if (platinfo.vendor.find("Advanced Micro") != std::string::npos ||
           platinfo.vendor.find("Advanced Micro") != std::string::npos ||
           platinfo.vendor.find("AMD") != std::string::npos)
  {
    if (device_name == vega_string())
    {
      wg_atom_size = 32;
    }

    else
    {
      wg_atom_size = 64;
    }
  }

Comment (zjing14) [reply miopengemm/src/oclutil.cpp]: line 1149 in miopengemm/src/oclutil.cpp

**Comment by zjing14** [comment]
the change of wg_atom_size from 32 to 64 barely affects performance.

**Pull request merged**